doc/nrf/libraries/security/nrf_security/doc/configuration.rst

```diff
@@ -16,7 +16,7 @@ PSA crypto support
 ******************
 
 PSA crypto support is included by default when you enable nRF Security through the :kconfig:option:`CONFIG_NRF_SECURITY` Kconfig option.
-PSA crypto support is provided through PSA Crypto APIs and is implemented by PSA core.
+PSA crypto support is provided through PSA Crypto version 1.2.1 APIs and is implemented by PSA core.
```

Comment:
@SebastianBoe , please update with link to documentation and correct version.

Reply:
fixed